src/flatten.jl

```diff
@@ -0,0 +1,58 @@
+function flatten(x::Real)
```

Comment:
is `flatten` supposed to have docstring?

Reply:
It should. Will write it.

Reply:
Written. Let me know if it's suffcient.

Reply:
yes, thanks